File: rest/change-call-state/list-get-example-2/list-get-example-2.5.x.rb
# Get twilio-ruby from twilio.com/docs/ruby/install
require 'twilio-ruby'

# Get your Account Sid and Auth Token from https://www.twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'

# Initialize Twilio Client
@client = Twilio::REST::Client.new(account_sid, auth_token)

@call = @client.account.calls('CAe1644a7eed5088b159577c5802d8be38').fetch

@call.update(
  status: 'completed'
)

puts @call.direction

File: rest/usage-records/list-get-example-2/list-get-example-2.5.x.php
<?php
// Get the PHP helper library from https://twilio.com/docs/libraries/php
require_once '/path/to/vendor/autoload.php'; // Loads the library
use Twilio\Rest\Client;

// Your Account Sid and Auth Token from twilio.com/user/account
$sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token = "your_auth_token";
$client = new Client($sid, $token);

$records = $client->usage
    ->records
    ->today
    ->read(
        array("category" => "calls")
    );

// Loop over the list of records and echo a property for each one
foreach ($records as $record) {
    echo $record->count;
}
